docs/tutorials/3_train_gail.ipynb

```diff
@@ -93,53 +89,58 @@
    "outputs": [],
    "source": [
     "from imitation.algorithms.adversarial.gail import GAIL\n",
-    "from imitation.rewards.reward_nets import BasicRewardNet\n",
+    "from imitation.rewards.reward_nets import BasicShapedRewardNet\n",
     "from imitation.util.networks import RunningNorm\n",
     "from imitation.util.util import make_vec_env\n",
     "from stable_baselines3 import PPO\n",
+    "from stable_baselines3.ppo import MlpPolicy\n",
     "from stable_baselines3.common.evaluation import evaluate_policy\n",
-    "from stable_baselines3.common.vec_env import DummyVecEnv\n",
-    "\n",
-    "import gym\n",
     "\n",
     "\n",
-    "venv = make_vec_env(\"seals/CartPole-v0\", n_envs=8, rng=rng)\n",
+    "env = make_vec_env(\"seals/CartPole-v0\", n_envs=8, rng=np.random.default_rng(SEED))\n",
```

Comment:
Why are we re-creating the environment rather than using existing `env`?

Reply:
it's indeed unnecessary. fixed now